walkthroughs/howto-http-headers/README.md

```diff
@@ -0,0 +1,82 @@
+# HTTP Header based routing example
+
+Clone this repository and navigate to the walkthrough/http-headers-and-priority folder, all commands will be ran from this location.
+
+## Service Mesh Details
+This mesh has the following resources:
+* A Virtual Service named `color.header-mesh.local`. 
+* A Virtual Router named `color-router` which serves as the provider for the virtual service above.
+* 8 Virtual Nodes:
+  1. `front-node`: Sends egress traffic to the `color.header-mesh.local` Virtual Service. It discovers instances via Cloud Map in the `front` service under the `header-mesh.local` namespace. i.e. the DNS name `front.$STACK_PREFIX-mesh.local`.
```

Comment:
nitpick: Given that each of this is "A Cloud Map Node for `color.header-mesh.local`", it would help readability and reduce verbosity to pull that description up. Fine as-is though.